Create 6.0.1-nuxeo-drive-release-notes.md (#2723)
nuxeo/doc.nuxeo.com-platform-spaces: src/client-apps/nuxeo-drive/nuxeo-drive-release-notes/src/client-apps/nuxeo-drive/nuxeo-drive-release-notes/6.0.1-nuxeo-drive-release-notes.md

---
title: 6.0.1
description: Release notes for Nuxeo Drive.
tree_item_index: 260
review:
  comment: 'release'

>   date: '2026-05-11'

Reviewer: 6.0.1 was released on '2026-02-23'
Pl. check

  status: ok
toc: true
---

Welcome to the Release Notes for **Nuxeo Drive 6.0.1**

**Status**: <font color="#0066ff">**Release**</font> </br>
<i class="fa fa-long-arrow-right" aria-hidden="true"></i> [Changelog](https://github.com/nuxeo/nuxeo-drive/blob/master/docs/changes/6.0.1.md)

## Core 

### End of Support macOS 13 

> Starting from Nuxeo Drive 6.0.1, macOS versions supported are >= 15, 64 bits. 

Reviewer 2: ```suggestion
Starting from Nuxeo Drive 6.0.1, macOS versions supported are version15 (64 bit) and higher. 
```

## Upgrade Behavior 

> We discovered an issue when upgrading from old Nuxeo Drive version to latest one, on Windows 11. The problem has to do with upgrade from 5.4.0 to 6.0.0. Upgrade from 5.5.x+ to 6.0.0 is unaffected. The issue has been fixed in Nuxeo Drive 6.0.1.

Reviewer 2: ```suggestion
We discovered an issue when upgrading from old Nuxeo Drive version to latest one, on Windows 11. The issue occurred when upgrading from 5.4.0 to 6.0.0. Upgrade from 5.5.x+ to 6.0.0 is unaffected. The issue has been fixed in Nuxeo Drive 6.0.1.
```
